app/models/user.js

```diff
@@ -0,0 +1,102 @@
+import Ember from "ember";
```

Comment:
I'm assuming you created this to avoid changing the model from `shared.goodcity` ?

Reply:
yes, there are many computed properties and associations which we dont need for donor app, so added it here.